db/seeds.rb

```diff
@@ -429,8 +429,8 @@ def generate_objs(method, obj, n)
 
```

Comment:
You should update the comment / heading above this section (Eqobj --> Eqitem, I guess?)